regression/scripts/Vehicles/Electric_Multicopter.py

```diff
@@ -253,5 +255,5 @@ def vehicle_setup():
     
     vehicle.append_component(net)
     
-    vehicle.weight_breakdown  = empty(vehicle,None)
+    #vehicle.weight_breakdown  = empty(vehicle)
```

Comment:
should this be commented out, or should it be deleted?

Reply:
Deleted.  I added it somewhere else. Thanks